docs/index.md

```diff
@@ -73,16 +68,16 @@ This starts a server on port 8080.
 
 #### Running with OpenFin
 
-OpenFin has a command line tool for launching OpenFin applications. This can be installed as follows:
+The ContainerJS project provides an OpenFin-based command line tool, which can be installed as follows:
```

Comment:
Is this actually true, or is this something we plan to do?

As far as I can tell the ssf-openfin command is available via the containerjs-api-openfin dependency (so you can run `ssf-openfin...` as a script in package.json), but you can't actually install it globally with that command. Same goes for ssf-electron actually.